test/unit/api/define.js

```diff
@@ -5,9 +5,9 @@ const { name: $name } = symbols;
 
 function mockDefine(name) {
   if (customElementsV1) {
-    window.customElements.define(name, function() {});
+    window.customElements.define(name, () => {});
```

Comment:
This is a perfect example of where Airbnb rules are just ludicrous. These functions require constructors (not arrow functions).